### PR TITLE
Sort output info to keep the right order.

### PR DESCRIPTION
**Pull request by XinWangIntel**
Sort output info to keep the right order.

### CODE REVIEW

File: ngraph/test/runtime/interpreter/evaluates_map.cpp [hunk 1]
```diff
@@ -694,7 +694,7 @@ namespace
         {
             if (box_encoding == V5BoxEncoding::CORNER)
             {
-                normalize_corner(boxes, boxes_shape);
+                // normalize_corner(boxes, boxes_shape);
```
Comment (tzerrell) [inline ngraph/test/runtime/interpreter/evaluates_map.cpp]: Why is this necessary? Or what happens if you don't do this? I'm concerned about whether this is implementing a necessary feature that we are unaware of.

File: inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp [hunk 2]
```diff
@@ -376,7 +376,23 @@ std::vector<std::vector<std::uint8_t>> LayerTestsCommon::CalculateRefs() {
...
+          const pp b) const {
+        bool result = false;
+        if (a.first.length() < b.first.length()) {
+          result = true;
+        } else if (a.first.compare(b.first) < 0) {
+          result = true;
+        }
+        return result;
+      }
+    } blobLess;
```
Comment (tzerrell) [inline inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp]: I'm a bit skeptical about the name `blobLess`, I'm not sure how "blob" is related to this comparison (perhaps I'm missing something?). I suggest `alphabeticalLess` or `lexicalLess` or `nameLess` instead.

**Review by tzerrell — COMMENTED** [review]
The deterministic ordering is reasonable enough -- when we go to upstream this we may find that a more semantic ordering is preferred but we can address that if & when it comes up.

I'm a little worried we might be destroying functionality from `normalize_corner`, but I haven't investigated... do you know what it does?

File: inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp (hunk 2 above)
Comment (XinWangIntel) [reply inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp]: Done, use nameLess, we want to compare output name here ( is blob),  for example , let the "xxx-8" to be placed before "xxx-10", but it has opposite order with traditional string comparison,  so nameLess may be better here.

File: ngraph/test/runtime/interpreter/evaluates_map.cpp (hunk 1 above)
Comment (XinWangIntel) [reply ngraph/test/runtime/interpreter/evaluates_map.cpp]: Yes, the reason to close and not implement it on our side is that it changes one box to another box so not meet the user's request.  

If we have a box which coordinate ([y1, x1, y2, x2]) is [1, 3, 4, 2] ,  the function will make it to [1, 2, 4, 3],  these two boxes are not equivalent, this means if we link two op, the output is not what the writer requires. 
It just looks like a workaround as the tests will produce random input data, but OPENVINO has no limitation here.  

And another function normalize_center() just converts [x, y, width, height] to [y1, x1, y2, x2], they are different expressions of the same box, so keep it.

Comment (tzerrell) [reply ngraph/test/runtime/interpreter/evaluates_map.cpp]: Do I understand correctly that this just changes the semantics of the box coordinates? And specifically, with this commented out, boxes provided in [y1, x1, y2, x2] order will be handled correctly and boxes in [y1, x2, y2, x1] order will be handled incorrectly? And, conversely, in the original form, boxes in [y1, x1, y2, x2] order will be handled incorrectly while boxes in [y1, x2, y2, x1] order will be handled correctly? Also, does commenting this out make the behavior of the reference implementation match the order specified in the docs?

If that understanding is correct, then we should make this change but also add a comment like `// TODO: Check if docs or original reference implementation is correct` so we know to come back to this during upstreaming (because it's theoretically possible the reference impl. is correct and the docs are wrong).

Or let me know if I misunderstood.

File: inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp (hunk 2 above)
Comment (tzerrell) [reply inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp]: Ah, thanks for the details, I didn't catch the reversed order. In that case, this name is fine, but reverseNameLess might be even better.

File: ngraph/test/runtime/interpreter/evaluates_map.cpp (hunk 1 above)
Comment (XinWangIntel) [reply ngraph/test/runtime/interpreter/evaluates_map.cpp]: My fault, convert [y1, x1, y2, x2] to [ymin, xmin, ymax, xmax] shall still be the same box, but for some special values need recheck, i will do more tests and info you later, thanks.

Comment (XinWangIntel) [reply ngraph/test/runtime/interpreter/evaluates_map.cpp]: Add https://github.com/plaidml/plaidml/pull/1757 to change box info to [ymin, xmin, ymax, xmax] in BoxesDecodeMode::NMS mode. This PR only do sort now.

Comment (tzerrell) [reply ngraph/test/runtime/interpreter/evaluates_map.cpp]: Sounds good, thanks!

**Review by tzerrell — APPROVED** [review]
LGTM
